suites/fs/multiclient/mount/fuse.yaml

```diff
@@ -0,0 +1 @@
+../../basic/mount/fuse.yaml
```

Comment:
In this particular case, it's just making the sub-suites consistent. Some sub-suites had the ceph-fuse task specified within a task (e.g. suites/fs/traceless/tasks/cfuse_workunit_suites_dbench.yaml) and others at the top-level (suites/fs/snaps/mount/ceph-fuse.yaml).